src/k8s/components/indexed-search/generate/service/indexed-search-indexer.dhall

```diff
@@ -49,9 +49,9 @@ let Service/generate
                 , name = Some "indexed-search-indexer"
                 }
               , spec = Some Kubernetes/ServiceSpec::{
+                , clusterIP = Some "None"
```

Comment:
This needs to be explicitly set, apparently

Reply:
yep, i see